dcap/types/Cargo.toml

```diff
@@ -30,7 +30,7 @@ serde = { version = "1.0.152", default-features = false, features = ["derive"],
 sha2 = { version = "0.10.6", default-features = false }
 static_assertions = "1.1.0"
 subtle = { version = "2.4.1", default-features = false }
-x509-cert = { version = "0.2.0", default-features = false, features = ["pem"], optional = true }
+x509-cert = { version = "0.2.3", default-features = false, features = ["pem"], optional = true }
```

Comment:
Any reason this isn't updated in dev-dependencies below as well?

Reply:
good catch!, will fix that

Reply:
updated the version in the dev-dependencies